### PR TITLE
fix(webpack): fix blank screen on hot reload [#24]

### PR DESCRIPTION
**Pull request by davidholyko**
#### Summary

- Fixes [#24]
- Fixed bug where an error would cause hot reloading to reload with a blank page

#### What I did

- Fixed eslint not showing errors
- Added eslint loader
- Fixed copy plugin to copy to build directory

#### Notes

- Copy Plugin was not working correctly
-
-

#### Before/After Preview

- Before:
- After:

#### Checklist

- [x] Commits are in semantic format
- [x] PR title is in format `TYPE(scope): description`
- [x] Branch is in format `TYPE/scope/description`
- [x] Has Summary

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -2,13 +2,14 @@
   "env": {
     "browser": true,
     "es6": true,
-    "react": true
```
Comment (davidholyko) [inline .eslintrc.json:5]: this didnt do anything

File: webpack.config.js [hunk 2]
```diff
@@ -4,16 +4,25 @@ const CopyPlugin = require('copy-webpack-plugin');
 
 module.exports = {
   mode: 'development',
-  entry: {
-    main: './src/index.js',
-  },
+  entry: './src/index.js',
```
Comment (davidholyko) [inline webpack.config.js:7]: removed chunking and splitting for now. The project is small enough that I dont think we need it

Comment (simonchen92) [reply webpack.config.js:7]: agreed

File: .eslintrc.json (hunk 1 above)
Comment (simonchen92) [reply .eslintrc.json:5]: I agree

**Pull request merged**